scripts/js/lib/api/processHtml.ts

```diff
@@ -339,6 +339,7 @@ export async function processMembersAndSetMeta(
   $: CheerioAPI,
   $main: Cheerio<any>,
   meta: Metadata,
+  isCApi: boolean,
```

Comment:
Nit to consider using `kwargs: {isCApi: boolean}` to make the code more readable. I think Arnau was going to make that change somewhere else but I'm not sure he's making the change anymore.

Not blocking!